### DISCUSSION TITLE
SCOS raw image

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#2034] [Q&A]
Hi.
I would like to install OKD, specifically in a bare-metal way. Which is the correct SCOS version for the specific OKD version, and where can I download the raw image?

**Comment by titou10titou10** [comment]
This comment from yesterday may be helpful https://github.com/okd-project/okd/discussions/2019#discussioncomment-10780600

**Comment by titou10titou10** [comment]
Not directly related to your question, but until bootstrap imaged are based on SCOS, I documented a (bumpy but no so hard) way to install the latest "stable" version of v4.17 and v4.16 with ABI/UPI/baremetal here: https://github.com/okd-project/okd/discussions/2035

**Comment by dkoci** [comment]
@titou10titou10 If I understand correctly, OKD-SCOS 4.16 is not stable and cannot be provisioned (without a workaround) on BM UBI, as described in the OKD 4.15 documentation?

**Comment by titou10titou10** [comment]
There is no GA version of OKD-SCOS at all (IPI/UPI cloud/BM...).
The latest GA version of OKD is v4.15.0-0.okd-2024-03-10-010116 (with FCOS)
